src/app/fddev/configure/genesis.c

```diff
@@ -114,6 +152,10 @@ init( config_t * const config ) {
     if( FD_UNLIKELY( WEXITSTATUS( wstatus ) ) )
       FD_LOG_ERR(( "genesis creation process exited with status %i", WEXITSTATUS( wstatus ) ));
   }
+
+  char genesis_path[ PATH_MAX ];
```

Comment:
I don't think this is a good place for this, it needs to happen regardless of if genesis is being generated on this run? fdctl doesn't run genesis, for example.

What if we don't have a genesis because we haven't connected to the chain yet? Anyway this should move to run.c probably.

Reply:
I moved it to dev.c. Logan is working on the other half of the problem.